index.d.ts

```diff
@@ -0,0 +1,105 @@
+// Type definitions for Aphrodite 1.2.3
+// Project: https://github.com/Khan/aphrodite
+
+declare module "aphrodite" {
+    import * as React from "react";
```

Comment:
I've been using TypeScript & Aphrodite for a side project, and had to use my own copy because I'm using Aphrodite but not react -- can you remove this, since Aphrodite doesn't have a dependency on React? As an initial pass, it should be fine to just replace `React.CSSProperties` with `any`.